crates/admin/src/cluster_controller/scheduler.rs

```diff
@@ -8,11 +8,9 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
-use rand::seq::IteratorRandom;
-use std::collections::{BTreeMap, BTreeSet};
-use tracing::{debug, trace};
-
+use crate::cluster_controller::logs_controller;
```

Comment:
Something, something imports ordering :-)

Reply:
Thanks for catching it. Will try out the nightly `rustfmt`.